README.md

```diff
@@ -148,7 +148,7 @@ For security, you need to enter something into the `SECRET_KEY`.
 SECRET_KEY = 'somethingnotempty'
 ```
 
-By default, you need a `Memcache` server up and running. The connection
+*By default, you need a `Memcache` server up and running.* The connection
```

Comment:
For code review changes, would you rather I add an additional commit with the fix or amend my previous commit?